generate.py

```diff
@@ -400,7 +401,8 @@ def generate(args):
     parser.add_argument('--include-price-csv-option', '-po', metavar=('KEY', 'VALUE'),
                         nargs=2, default=[], action='append',
                         help='append additional argument to price signals')
-    parser.add_argument('--config', help='Use config file to set arguments')
+    parser.add_argument('--config', help='Use config file to set arguments',
+                        default="examples/generate.cfg")
```

Comment:
I don't like that. The config file is a major change to all arguments. It should only be set actively, not because this argument is not set. Your change makes it much easier to unwittingly use the config, much harder to not use a config at all and therefore more likely to generate unexpected results (more so, as you are not warned when overriding arguments by providing a config file). It is cleaner to set the necessary options yourself instead of relying on the content of a file that might have been changed. Use the argparse Namespace functionality, there is a reason why parsing the arguments and the functionality are separated.

Reply:
ok!